test/Dialect/SSP/Transforms/print.mlir

```diff
@@ -5,6 +5,7 @@ ssp.instance @Foo of "Problem" {
   library {
     operator_type @Bar [latency<1>]
   }
+  resource {}
```

Comment:
Can we change the printing so the resource section does not print if it is empty?

Reply:
But the empty `library`/`graph` are still being printed:
https://github.com/llvm/circt/blob/c07d31b33912b2b33d36f1f8f6312b0a4936137a/test/Dialect/SSP/roundtrip.mlir#L8-L11